Backpack/src/main/res/values/backpack.animation.xml

```diff
@@ -17,6 +17,9 @@
   limitations under the License.
 -->
 <resources>
+
+  <!-- THIS FILE IS AUTO GENERATED. DO NOT MODIFY! -->
```

Comment:
this was in the text file, but not any others - thought i'd include it in the template for all to make it more obvious what's autogenerated vs not